.gitignore

```diff
@@ -53,4 +53,8 @@ examples/
 build/dist
 scripts/aws/aws
 controllers/sriov-controller/sriov-controller
-cmd
\ No newline at end of file
+cmd
+!cmd/
+!build/dist/
+!scripts/aws/aws/
+!controllers/sriov-controller/sriov-controller/
```

Comment:
this allows tracking new files in a path.
For example,`cmd` binary is ignored, but all files in path */cmd/* are not ignoring.